### PR TITLE
Automate the version change of Express

### PR DESCRIPTION
**Pull request by bjohansebas**
This script automates the process of changing the version number in the documentation, see #1504.

To ensure the workflow functions correctly, please review #1606

### CODE REVIEW

**Review by UlisesGascon — APPROVED** [review]

File: get-express-version.mjs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { readFile } from 'node:fs/promises'
+import path from 'node:path'
+
+const NPMURL = 'https://registry.npmjs.org/express'
+
+const response = await fetch(NPMURL).then((res) => res.json())
+
+const { next, latest } = response['dist-tags']
+
+const contentFile = await readFile(
```
Comment (IamLizu) [inline get-express-version.mjs]: Should we add error handling here?

File: get-express-version.mjs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  path.resolve(path.join('_data', 'express.yml')),
+  'utf8'
+)
+  .then((content) =>
```
Comment (IamLizu) [inline get-express-version.mjs]: It's better to use `await` consistently instead of mixing with `.then()`.

File: get-express-version.mjs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { writeFileSync } from 'node:fs'
+import { readFile } from 'node:fs/promises'
+import path from 'node:path'
+
+const NPMURL = 'https://registry.npmjs.org/express'
+
+const response = await fetch(NPMURL).then((res) => res.json())
```
Comment (IamLizu) [inline get-express-version.mjs]: It's better to use `await` consistently instead of mixing with `.then()`.

**Review by IamLizu — CHANGES_REQUESTED** [review]
Hey @bjohansebas 👋 

Great job on automating this 💯, I am generally 👍 on this. However, would you be open to something like this? Perhaps we should make consistent use of `await`.

I have also added error-handling just to _feel safe_.
```mjs
import { readFile, writeFile } from 'node:fs/promises';
import path from 'node:path';

const NPMURL = 'https://registry.npmjs.org/express'

const response = await (await fetch(NPMURL)).json();
const { next, latest } = response['dist-tags']

try {
  const filePath = path.resolve(path.join('_data', 'express.yml'));
  let content = await readFile(filePath, 'utf8');
  
  content = content.replace(/current_version: ".*"/, `current_version: "${latest}"`);
  content = content.replace(/next_version: ".*"/, `next_version: "${next}"`);
  
  await writeFile(filePath, content, 'utf8');
} catch (error) {
  console.error('Error updating versions in _data/express.yml:', error);
}
```

**Review by juliogarciape — APPROVED** [review]

**Comment by bjohansebas** [comment]
Thanks @IamLizu for the suggestions.

File: get-express-version.mjs (hunk 1 above)
Comment (bjohansebas) [reply get-express-version.mjs]: done, thanks

File: get-express-version.mjs (hunk 2 above)
Comment (bjohansebas) [reply get-express-version.mjs]: done, thanks

File: get-express-version.mjs (hunk 3 above)
Comment (bjohansebas) [reply get-express-version.mjs]: done, thanks

**Review by carlosstenzel — APPROVED** [review]

**Pull request merged**